Makefile

```diff
@@ -1,47 +1,48 @@
-PKG=github.com/getstream/easyjson
-GOPATH:=$(PWD)/.root:$(GOPATH)
+PKG=github.com/GetStream/easyjson
+GOPATH:=$(PWD)/_root:$(GOPATH)
 export GOPATH
+export GO111MODULE=off
```

Comment:
Why are you disabling modules here?

Reply:
I'm not 100% sure but builds were failing and I believe it was due to the fact that the make file is building a small temporary GOPATH environment. Then the modern go compiler was trying to ignore the GOPATH and treat everything like it was a go module. Turning go modules off let it fallback to the GOPATH behaviour.